### ISSUE TITLE
Networkgraph color bug

### ISSUE THREAD

**Issue by Vicroline** [jbkunst/highcharter#506]
First of all, highcharter is awesome thanks for providing and maintaining the package!

I don't know why but if a network graph has more than 40 nodes the group color values are not working properly. You can reproduce this by increasing N in your sample:

```
library("highcharter")
library("igraph")
# up until N <- 40 it works just fine
N <- 100 

net <- sample_gnp(N, p = 2/N)
wc <- cluster_walktrap(net)

V(net)$label <- seq(N)
V(net)$name <- paste("I'm #", seq(N))
V(net)$page_rank <- round(page.rank(net)$vector, 2)
V(net)$betweenness <- round(betweenness(net), 2)
V(net)$degree <- degree(net)
V(net)$size <- V(net)$degree
V(net)$comm <- membership(wc)
V(net)$color <- colorize(membership(wc))

hchart(net, layout = layout_with_fr)
```

**Comment by Patanouque** [comment]
Hello,
I'm having the same problem, it definitely looks like a bug
I tried to track down the problem and it comes from the last line of hchart.igraph :
```
hc <- hc %>%
    hc_add_series(data = NULL, name = "edges", id = "e") %>% 
    hc_add_series_list(dse) # the bug happens when this serie is added 
```

I don't know how to solve it and this behaviour is not consistent (sometimes the bug happens at N=50, sometime above or below).

The black dots that appear is the serie "labels" which for some bizarre reasons become a scatter plot when N is high...

Maybe the problem comes from highcharts.js...

Any help on this would be much appreciated !

**Comment by jbkunst** [comment]
Hi @Vicroline 

Solved in #586
